platform/docs/src/main/resources/_contents/extcontents.adoc

```diff
@@ -112,3 +112,30 @@ The following is a sample taxonomy: 
 |Get Metadata
 
 |===
+
+=== Developing Migratables
+
+The `Migratable` API provides a mechanism for bundles to handle exporting data required to clone a {branding} system.
+The migration process is meant to be flexible, so an implementation of `org.codice.ddf.migration.Migratable` can handle
+exporting data for a single bundle or groups of bundles such as applications. For example, the
+`org.codice.ddf.platform.migratable.impl.PlatformMigratable` handles exporting core system files for the {branding} Platform Application.
+Exporting configurations stored in `org.osgi.service.cm.ConfigurationAdmin` does not need to be handled by implementations of
+`org.codice.ddf.migration.Migratable` as all `ConfigurationAdmin` configurations are exported by
+`org.codice.ddf.configuration.admin.ConfigurationAminMigration`.
+
+The Migratable API includes:
+
+. `org.codice.ddf.migration.Migratable`
+. `org.codice.ddf.migration.AbstractMigratable`
+. `org.codice.ddf.migration.MigrationException`
+. `org.codice.ddf.migration.MigrationMetadata`
+. `org.codice.ddf.migration.MigrationWarning`
+
+==== Usage
+
+In order to create a `Migratable` for a module of the system, the `org.codice.ddf.migration.Migratable` interface must be implemented
+and the implementation must be registered under the `org.codice.ddf.migration.Migratable` interface as an OSGI service in
+the OSGI service registry. Creating an OSGI service allows for the `org.codice.ddf.configuration.migration.ConfigurationMigrationManager`
+to lookup all implementations of `org.codice.ddf.migration.Migratable` and command them to export. The abstract base class
+`org.codice.ddf.migration.AbstractMigratable` in the `platform-migratable-api` implements common boilerplate code required
+when implementing `org.codice.ddf.migration.Migratable` and should be extended when creating a `org.codice.ddf.migration.Migratable`.
```

Comment:
Good stuff!

Other topics we may want to consider:
* When to use `MigrationWarnings` vs. `MigrationException`.
* How `getDescription()` and `isOptional()` will be used.
* Where files will be written to.
* The fact that `Migratable` implementations should create their own sub-directory if the files they export shouldn't end up in the `etc` directory, or to avoid file name clashes.

Anything else?